entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,361 @@
+<div class="body body_bg">
+    <div class="container container_bg">
+        <div class="main">
+            <div class="user">
```

Comment:
Request to use these required components and update the application.
https://github.com/applibgroup/Theme
https://github.com/applibgroup/Alert
https://github.com/applibgroup/Buttons
https://github.com/applibgroup/Card
https://github.com/applibgroup/Checkbox
https://github.com/applibgroup/Dropdown
https://github.com/applibgroup/form
https://github.com/applibgroup/Navbar
https://github.com/applibgroup/pagination
https://github.com/applibgroup/progress